applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/status.js

```diff
@@ -0,0 +1,267 @@
+				E('th', { 'class': 'th' }, _('roam tgt')),
+				E('th', { 'class': 'th' }, _('rrm_nr mac')),
+				E('th', { 'class': 'th' }, _('rrm_nr hex'))
+				
+			])
+		]);
+	
+		var connectioninfo_table_entries =[];
+		for(var wlan in Localinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Localinfo[wlan]['bssid'],
+							Localinfo[wlan]['ssid'],
+							Localinfo[wlan]['freq'],
+							Localinfo[wlan]['n_assoc'],														
+							Localinfo[wlan]['noise'],														
+							Localinfo[wlan]['load'],														
+							Localinfo[wlan]['max_assoc'],														
+							Localinfo[wlan]['roam_events']['source'],														
+							Localinfo[wlan]['roam_events']['target'],														
+							Localinfo[wlan]['rrm_nr'][0],																					
+							Localinfo[wlan]['rrm_nr'][2]		
+						]);
+				}	
+		for(var wlan in Remoteinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Remoteinfo[wlan]['bssid'],
+							Remoteinfo[wlan]['ssid'],
+							Remoteinfo[wlan]['freq'],
+							Remoteinfo[wlan]['n_assoc'],														
+							Remoteinfo[wlan]['noise'],														
+							Remoteinfo[wlan]['load'],														
+							Remoteinfo[wlan]['max_assoc'],														
+							Remoteinfo[wlan]['roam_events']['source'],														
+							Remoteinfo[wlan]['roam_events']['target'],														
+							Remoteinfo[wlan]['rrm_nr'][0],																					
+							Remoteinfo[wlan]['rrm_nr'][2]		
+
+						]);
+				}	
+		cbi_update_table(connectioninfo_table, connectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(connectioninfo_table);
+/////////////////////////////////////////
+
+
+		var compactconnectioninfo_table = E('table', { 'class': 'table cbi-section-table' }, [
+				E('tr', { 'class': 'tr table-titles' }, [
+					E('th', { 'class': 'th' }, _('wlan')),
+					E('th', { 'class': 'th' }, _('ssid')),
+					E('th', { 'class': 'th' }, _('freq')),
+					E('th', { 'class': 'th' }, _('load')),
+					E('th', { 'class': 'th' }, _('n')),
+					E('th', { 'class': 'th' }, _('host'))
+					
+				])
+			]);
+		var compactconnectioninfo_table_entries =[];
+		for(var wlan in Localinfo) {
+			var hostl=''
+			for(var mac in Clients) {
+				if (typeof Clients[mac] !== 'undefined') 
+					if (typeof Clients[mac][wlan] !== 'undefined') 
+					if (String(Clients[mac][wlan]['connected']).valueOf()==String("true").valueOf()) {
+							var foundname=mac;
+							var macUp=String(mac).toUpperCase()
+							if (typeof hosts[macUp] !== 'undefined') {
+								if ((String(hosts[macUp]['ipaddrs'][0]).length>0) &&
+									(typeof hosts[macUp]['ipaddrs'][0] !== 'undefined')) 
+										foundname=hosts[macUp]['ipaddrs'][0];
+								if ((String(hosts[macUp]['name']).length>0) &&
+									(typeof hosts[macUp]['name'] !== 'undefined')) 
+										foundname=hosts[macUp]['name'];
+								}
+							hostl=hostl+ foundname+'&emsp;';
+				}
```

Comment:
I don't know what are the rules for luci/OpenWrt, but I usually add brackets always, despite it has only one sentence inside, and the indentation is wrong.